Podcast/SearchTableViewController.swift

```diff
@@ -73,9 +72,11 @@ class SearchTableViewController: ViewController, UITableViewDelegate, UITableVie
         tableView.delegate = self
         tableView.dataSource = self
         tableView.infiniteScrollIndicatorView = createLoadingAnimationView()
+        tableView.tableFooterView = UIView()
```

Comment:
reason for this?

Reply:
a lot of the stuff i did in searchtableviewcontroller was to see if i could figure out the inset bug, so i was trying to add a footerview to bump the tableview upwards past the tab bar